.github/workflows/ci.yml

```diff
@@ -32,6 +32,10 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: 2.6
+      - name: Add PHP for a test that requires it
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.3'
```

Comment:
Just pointing out that I think this is odd and I'll look into it further unless you do.

I thought php was installed in GH Actions... so maybe that changed?

Reply:
It was very strange and it only seemed to happen on MacOS. If you want to check it out, I ran the [CI tests](https://github.com/jpal91/mask/actions/runs/9022227730) on my forked repo, but this was the output -

```
command=`CLICOLOR_FORCE="1" name="World" "/Users/runner/work/mask/mask/target/debug/mask" "--maskfile" "/var/folders/3m/p59k4qdj0f17st0gn2cmj3640000gn/T/.tmpDcfjK1/maskfile.md" "php"`
code=1
stdout=```""```
stderr=```"\u{1b}[31mERROR:\u{1b}[0m program \'php\' for executor \'php\' not in PATH\n"```
```

The Windows failure was my fault, I messed up defining the params in a powershell command, so you can ignore that one. 

I tried to do a little searching online, but couldn't come up with anything concrete. It's just really weird...

Reply:
So because I use macos-latest, and it's been awhile, that runner image is now macOS 14 instead of [13 which used to come with php](https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md).

This [table](https://github.com/actions/runner-images/tree/main?tab=readme-ov-file#available-images) shows there's 2 variations of macOS-latest:
 
<img width="880" alt="Screenshot 2024-05-16 at 3 41 18 PM" src="https://github.com/jacobdeichert/mask/assets/1631044/beb038fc-82ab-4ae1-8793-c8d4bf25f9f7">


* [macos-latest](https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md) - does not come pre-installed with php
* [macos-latest-large](https://github.com/actions/runner-images/blob/main/images/macos/macos-14-Readme.md) - DOES come pre-installed with php

So the solution here is to change this workflow to reference `macos-latest-large` instead of `macos-latest` and this will ensure php is available.

Can you do this and remove this setup-php action too?

Reply:
So I did update this in my most recent commit. However, I couldn't run the actions in my fork because of the core limit on my pro subscription doesn't cover the `large` packages since they're over 4 cores, I believe.

I'm not sure why the CI isn't running on your side, though. I'm just seeing that it needs reviewer approval first?

Reply:
Ah, I didn't know the larger image requires a paid plan!

I see they are failing and I'll look into it further today. I'll probably make a PR to master so you don't have to worry about this after your branch updates.

Reply:
Now we know holding php hostage is what gets GitHub the big money :laughing:. 

Sounds good but if you want me to add in another runner, just let me know, it's not a problem.

Reply:
I didn't get around to this yesterday unfortunately.

Now I'm on vacation for the next week, but planning on looking at this again once I'm back 👍

Reply:
No worries, enjoy the vacation!

Reply:
Thanks! I fixed the php issue in PR https://github.com/jacobdeichert/mask/pull/110. Once that merges, please update your branch with master whenever you find the time (you'll have conflicts).

I'm aiming to give this PR a proper review within the next week or so. Apologies for the delays!

Reply:
Sorry that took so long! Last couple of weeks have been busy.

I went ahead and merged. It looks like the new runner works, so we should be good to go!